app/viewmodels/checkAnswers/SelectReasonSummary.scala

```diff
@@ -26,27 +27,32 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
-object SelectReasonSummary  {
+import javax.inject.Inject
 
-  def row(userAnswers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
-    userAnswers.get(SelectReasonPage).map {
+class SelectReasonSummary @Inject()()  {
+
+  def row(alertOrReject: SelectAlertReject)(implicit request: DataRequest[_], messages: Messages): Option[SummaryListRow] =
+    request.userAnswers.get(SelectReasonPage).map {
       answers =>
 
         val value = ValueViewModel(
           HtmlContent(
             answers.map {
               answer => HtmlFormat.escape(messages(s"selectReason.$answer")).toString
             }
-            .mkString(",<br>")
+            .mkString("<br><br>")
```

Comment:
As mentioned, I wonder whether this should be something like [this](https://github.com/hmrc/emcs-tfe-explain-shortage-excess-frontend/pull/48/files) to match the prototype?

Reply:
fixed with commit https://github.com/hmrc/emcs-tfe-alert-rejection-frontend/pull/22/commits/fcb47366a5ef537a879e9b6800c787bd39292f10